src/textnote.ts

```diff
@@ -112,26 +71,29 @@ export class TextNote extends Note {
     super(noteStruct);
 
     this.text = noteStruct.text || '';
-    this.superscript = noteStruct.superscript;
-    this.subscript = noteStruct.subscript;
-    this.setFont(noteStruct.font);
+    if (noteStruct.glyph) {
+      this.text += TextNote.GLYPHS[noteStruct.glyph] || noteStruct.glyph;
+    }
+    if (noteStruct.font) this.setFont(noteStruct.font);
+    else if (noteStruct.glyph == undefined) this.setFont(Tables.lookupMetricFontInfo('TextNote.text.fontSize'));
```

Comment:
We should try to avoid double equals == in our code.

Here we can use `noteStruct.glyph === undefined` or just write `!noteStruct.glyph` if you like.

Reply:
Also, for this if/else if/ else, I'd prefer to use braces { } even though each case is a single line. It makes it easier to read/skim in my opinion.

Reply:
done